docs/content/2.deploy/20.providers/cloudflare.md

```diff
@@ -22,6 +22,10 @@ compatibility_date = "2022-09-10"
 account_id = "<the account_id you obtained (optional)>"
 route = "<mainly useful when you want to setup custom domains (optional too)>"
 
+rules = [
+  { type = "ESModule", globs = ["**/*.js", "**/*.mjs"]},
+]
+
```

Comment:
I've added those here to simplify the documentation and not requiring people to set them only when using the cloudflare_module preset.

The the service worker output these are not going to do anything but they are not breaking anything either (as you can see from this [example](https://github.com/dario-piotrowicz/nitro-nuxt-lazy-loading-test/tree/main/nitro-no-lazy-loading-workers-test)), that's why I think it's simpler to just include them here, but let me know what you think, I don't mind moving it into the cloudflare_module preset section.